devlab/tests/config.py

```diff
@@ -34,35 +39,46 @@
               'to_port': '80',
               'cidr': '0.0.0.0/0'}]
           },
-         {'name': 'sg12', 'description': 'Blah blah group2'}]
+         {'name': 'sg12', 'description': 'Blah blah group2'}],
+     'cinder_volumes': [
+         {'name': 'tn1_volume1', 'size': 1, 'server_to_attach': 'tn1server1',
+          'device': '/dev/vdb'},
+         {'name': 'tn1_volume2', 'size': 1}
+     ],
+     'cinder_snapshots': [
+         # {'name': 'tn1snapsh', 'volume_id': 'tn1_volume2'}
```

Comment:
Let's put a comment describing why snapshot creation is disabled in our test scenario